_data/companies.yml

```diff
@@ -1503,8 +1503,8 @@
   events: Discouraged
   last_update: 2020-03-12
 
-- name: Urjanet
-  wfh: Optional
+- name: Lendis
+  wfh: Required
```

Comment:
This is deleting Urjanet from the list of companies, which we don't want to do.

Reply:
oh, sry my bad, fixing it

Reply:
@drewbrew fixed